apps/dot-voting/app/app-state-reducer.js

```diff
@@ -0,0 +1,7 @@
+function appStateReducer(state) {
```

Comment:
What if state is null?

Reply:
With removing appStateReducer from the equation this question became answered (indirectly).